tests/python_client/testcases/test_utility.py

```diff
@@ -1558,7 +1557,6 @@ def test_load_balance_normal(self):
         assert set(sealed_segment_ids).issubset(des_sealed_segment_ids)
 
     @pytest.mark.tags(CaseLabel.L1)
-    @pytest.mark.xfail(reason="need newer SDK")
```

Comment:
so we don't need to add @pytest.mark.xfail(reason="need newer SDK").
Because in CI , the version of pymilvus will be updated to the version in requirements.txt

Reply:
this pr removes xfail, not add them

Reply:
my mistake!